src/panels/ActionsPanel/ActionsPanel.tsx

```diff
@@ -41,12 +40,8 @@ export default function ActionsPanel(): JSX.Element {
   const appStore = useAppStore();
   const appState = appStore.getState();
   const { appName, modelName, userName } = useParams<EntityDetailsRoute>();
-  const getModelUUIDMemo = useMemo(
-    () => (modelName ? getModelUUID(modelName) : null),
-    [modelName],
-  );
-  const modelUUID = useSelector((state: RootState) =>
-    getModelUUIDMemo?.(state),
+  const modelUUID = useAppSelector((state: RootState) =>
+    modelName ? getModelUUID(state, modelName) : null,
```

Comment:
Same here about support a null model name in getModelUUID so you can remove this ternary.